centos_install_requirements.sh

```diff
@@ -32,7 +32,7 @@ sudo yum -y install \
   wget
 
 # We're reusing some tripleo pieces for this setup so clone them here
-pushd $HOME
+pushd "$HOME"
```

Comment:
It looks like all the changes in this file can just be removed, or split into another commit at least.